### DISCUSSION TITLE
Create QR Code for this page context menu item removed from 110.0.1587.41

### DISCUSSION THREAD

**Discussion by q71114** [MicrosoftEdge/WebView2Feedback#4020] [Q&A]
Is this a regression or was it removed deliberately? 

Our users rely on this functionality so it would be good to know if we need to implement this ourselves or not.

In `109.0.1518.78` the menu item was still there.
![image](https://user-images.githubusercontent.com/8770144/218430903-f17a16e7-45d2-4f98-90ee-dc1e2f837884.png)
In `110.0.1587.41` it got removed
![image](https://user-images.githubusercontent.com/8770144/218431041-cb18199d-a13e-418e-b01d-4927e2ed4fb4.png)

**Comment by novac42** [comment]
Thanks for reaching out. We've assigned it to a dev who can best answer the question.

**Comment by q71114** [comment]
@lflores-ms Did you have the chance to look at this?
